packages/hooks/src/__tests__/useIsRealAccountNeededForCashier.spec.tsx

```diff
@@ -0,0 +1,79 @@
+import * as React from 'react';
+import { StoreProvider } from '@deriv/stores';
+import type { TStores } from '@deriv/stores';
+import { renderHook } from '@testing-library/react-hooks';
+import useIsRealAccountNeededForCashier from '../useIsRealAccountNeededForCashier';
+
+describe('useIsRealAccountNeededForCashier', () => {
+    test('should return false if user is not in real account', async () => {
+        const mockRootStore: DeepPartial<TStores> = {
+            traders_hub: {
+                is_real: false,
+            },
```

Comment:
@thisyahlen-deriv Nice 🚀 You can use the `mockStore` method from `@deriv/stores` so you won't need to provide default values for mock data.